packages/features/ee/workflows/pages/index.tsx

```diff
@@ -43,7 +43,7 @@ function WorkflowsPage() {
       title={t("workflows")}
       subtitle={t("workflows_to_automate_notifications")}
       CTA={
-        session.data?.hasValidLicense && data?.workflows && data?.workflows.length > 0 ? (
+        session.data?.license.valid && data?.workflows && data?.workflows.length > 0 ? (
```

Comment:
License validity within the session object has been moved to its own object together with the key in order to have it at hand for other usages.